source/module_hsolver/hsolver_pw.cpp

```diff
@@ -217,15 +217,15 @@ void HSolverPW<T, Device>::solve(hamilt::Hamilt<T, Device>* pHamilt,
 
             GlobalC::paw_cell.set_paw_k(npw,
                                         wfc_basis->npwk_max,
-                                        kpt,
+                                        &kpt[0],
```

Comment:
feed what returned by `.data()` instead of using `&`. The memory address of `std::vector` is in principle dynamic, most of time should forget about memory address of std::vector